File: Python/is_mirror.py
import re

def is_mirror(str1, str2):
    
    clean1 = re.sub(r'[^A-Za-z]', '', str1)
    clean2 = re.sub(r'[^A-Za-z]', '', str2)
    
    
    reversed_str1 = clean1[::-1]
    
    
    return reversed_str1 == clean2 and clean1 != clean2

File: JavaScript/isMirror.js
function isMirror(str1, str2) {
  
  const clean1 = str1.replace(/[^A-Za-z]/g, '');
  const clean2 = str2.replace(/[^A-Za-z]/g, '');
  
  
  const reversed = clean1.split('').reverse().join('');
  
  
  return reversed === clean2 && clean1 !== clean2;
}